board-viewer.html

```diff
@@ -2416,8 +2445,12 @@
       breaks: false
     });
 
+    document.addEventListener('keydown', this._onMultiSelectEntering.bind(this));
+    document.addEventListener('keyup', this._onMultiSelectLeaving.bind(this));
+
     if (this.hasAttribute("board")) {
       this.loadBoard(this.getAttribute("board"));
+      this.$.viewer.focus();
```

Comment:
The board should not steal focus from the hosting app. Let the browser's tabindex or user clicking set focus.